hail/python/hail/experimental/db.py

```diff
@@ -78,22 +192,29 @@ class DB:
     This class facilitates the annotation of genetic datasets with variant
     annotations. It accepts either an HTTP(S) URL to an Annotation DB
     configuration or a python :obj:`dict` describing an Annotation DB
-    configuration.
+    configuration. User must specify the region ('us' or 'eu') in which the
+    cluster is running if connecting to the default Hail Annotation DB.
 
     Examples
     --------
     Create an annotation database connecting to the default Hail Annotation DB:
 
-    >>> db = hl.experimental.DB()
+    >>> db = hl.experimental.DB(region='us')
     >>> mt = db.annotate_rows_db(mt, 'gnomad_lof_metrics') # doctest: +SKIP
     """
 
     _valid_key_properties = {'gene', 'unique'}
+    _valid_regions = {'us', 'eu'}
 
     def __init__(self,
                  *,
+                 region=None,
```

Comment:
So, this breaks interface backwards compatibility. Do you see an issue with giving a default argument of `'us'`?

Reply:
I don't think it will be an issue. Should there be a warning or something when creating a `DB` object where `region='us'`? Or just make it clear in the docs that it will default to the 'us' region if 'eu' is not otherwise specified?